app/app.go

```diff
@@ -175,33 +175,8 @@ var (
 	// ModuleBasics defines the module BasicManager is in charge of setting up basic,
 	// non-dependant module elements, such as codec registration
 	// and genesis verification.
-	ModuleBasics = module.NewBasicManager(
-		auth.AppModuleBasic{},
-		genutil.AppModuleBasic{},
-		bank.AppModuleBasic{},
-		capability.AppModuleBasic{},
-		staking.AppModuleBasic{},
-		mint.AppModuleBasic{},
-		distr.AppModuleBasic{},
-		gov.NewAppModuleBasic(getGovProposalHandlers()...),
-		params.AppModuleBasic{},
-		crisis.AppModuleBasic{},
-		slashing.AppModuleBasic{},
-		feegrantmodule.AppModuleBasic{},
-		upgrade.AppModuleBasic{},
-		evidence.AppModuleBasic{},
-		authzmodule.AppModuleBasic{},
-		ibc.AppModuleBasic{},
-		transfer.AppModuleBasic{},
-		vesting.AppModuleBasic{},
-		ica.AppModuleBasic{},
-		icactlmodule.AppModuleBasic{},
-		evm.AppModuleBasic{},
-		feemarket.AppModuleBasic{},
-		gravity.AppModuleBasic{},
-		// this line is used by starport scaffolding # stargate/app/moduleBasic
-		cronos.AppModuleBasic{},
-	)
+	// Contains experimental modules by default.
+	ModuleBasics = GenModuleBasics(true)
```

Comment:
should we set the default to false? for keeping the logic consistent
the default `ExperimentalFlag` is false,

Reply:
The default one is for the places that don't have access to the experimental flag, I only changed the several cmds to add the flag and create a more specific module manager.

Reply:
So here might have an issue because the flag couldn't change the `ModuleBasics `
https://github.com/crypto-org-chain/cronos/blob/1573a354f6378b634bdf3702cf74f93dc9f50f06/app/test_helpers.go#L87

Reply:
this is for unit tests, so should be fine?

Reply:
and here
https://github.com/crypto-org-chain/cronos/blob/1573a354f6378b634bdf3702cf74f93dc9f50f06/testutil/network/network.go#L65

both for the unit tests, it should be fine, maybe add some comments saying the tests be involved with the experimental feature?

Reply:
> this is for unit tests, so should be fine?

in the `setup` function, there is an input argument for the `ExperimentalFlag` that might make the test behavior not meet the expectation.
https://github.com/crypto-org-chain/cronos/blob/1573a354f6378b634bdf3702cf74f93dc9f50f06/app/test_helpers.go#L76

Reply:
that makes sense, changed.
and `testutil/network/network.go` is removed, not used anywhere.